utest/variables/test_variables.py

```diff
@@ -15,12 +15,16 @@
 # Simple objects needed when testing assigning objects to variables.
 # JavaObject lives in '../../acceptance/testdata/libraries'
 
-class PythonObject:
+class PythonObject(object):
     def __init__(self, a, b):
         self.a = a
         self.b = b
+    def __getitem__(self, index):
+        return (self.a, self.b)[index]
     def __str__(self):
         return '(%s, %s)' % (self.a, self.b)
+    def __len__(self):
+        return (self.a, self.b).__len__()
```

Comment:
Why not just `return 2` here?

Reply:
Good point, changed by 898921a8052d058bec881ee807e3585c584819a1